CHANGELOG.md

```diff
@@ -80,8 +80,23 @@ All notable changes for each version of this project will be documented in this
         Use `IgxGridToolbarComponent`, `IgxGridToolbarHidingComponent`, `IgxGridToolbarPinningComponent` instead.
     - **Breaking Change** - The `rowSelected` event is renamed to `rowSelectionChanging` to better reflect its function.
     - **Breaking Change** - The `columnSelected` event is renamed to `columnSelectionChanging` to better reflect its function.
-- `igxGrid`
+    - **Breaking Change** - `columnsCollection` is removed. Use `columns` instead.
     - Exposed a `groupStrategy` input that functions similarly to `sortStrategy`, allowing customization of the grouping behavior of the grid. Please, refer to the [Group By ](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/groupby) topic for more information.
+- `RowType`, `IgxRowDirective`
+    - **Breaking Change** - `rowData` and `rowID` deprecated properties are now removed. Use `data` and `key` instead. Use `ng update` for automatic migration.
+- `RowSelectorTemplate`
```

Comment:
Kind of hard to tell from the diff, but these bullets don't seem to be under the grids as nested level, yet they probably should be, right?

Also either "`igxRowSelector` template" or "Row Selector template" - pick one, cuz the code highlighted `RowSelectorTemplate` is not a class/selector/anything we have.
..Though looking at the other sections I don't blame you, it's a bit of a mess